prov/verbs/src/fi_verbs.c

```diff
@@ -1990,7 +1990,7 @@ fi_ibv_eq_read(struct fid_eq *eq, uint32_t *event,
 
 	ret = rdma_get_cm_event(_eq->channel, &cma_event);
 	if (ret)
-		return (errno == EAGAIN) ? 0 : -errno;
+		return (errno == EAGAIN) ? -FI_EAGAIN : -errno;
```

Comment:
That would return EAGAIN instead of FI_EAGAIN. What is the requirement w.r.t. returning error codes? I was of the understanding that the provider should use FI_ error codes to report any error. I was planning to open a separate PR to convert all regular error codes to FI_ error codes.